src/lib/bitbucket-server/bitbucket-server-contributors.ts

```diff
@@ -179,3 +201,114 @@ export const fetchBitbucketReposForProjects = async (
   }
   return repoList;
 };
+
+const createImportFile = async (
```

Comment:
this function should probably be moved to be defined only once outside of SCM specific code, at least for most of it like capturing user's input and such.

Reply:
done